elastic/integrations
# Redis enterprise Integration- Beta package

## What does this PR do?

Redis Enterprise Integration beta release. Closes #3457

## Checklist


## Pending Items

- System test not planned refer ( [system-test-note.md](https://github.com/elastic/integrations/compare/main...lalit-satapathy:redis-enterprise-ver1?expand=1#diff-5742b68834ef31eb25b2f044b5885bfe7c3661aa8fe73988dd292d955471fa8e) )

## How to test this PR locally

- Docker install redislabs/redis:5.2.2-24
- Create a dummy node and database
- Ensure that the prometheus port (8070) is exported
- Pass the host:8070 to the integration hosts

## Related issues

- Closes #3457

## Screenshots

<img width="2544" alt="redisenterprise-dashboard" src="https://user-images.githubusercontent.com/69236064/177478189-55ef5691-3ca1-43ab-b522-afeef0ebdec6.png">
